src/scripts/components/main/menu/menu.styl

```diff
@@ -80,15 +80,16 @@
   font-size: 10px
 
   svg
-    height: emCalc(30px)
-    fill: $textDefault
+    height: remCalc(25px)
```

Comment:
why rem? if the parent has `font-size: 10px` you can also use `emCalc(25px, 10)`

Reply:
Why do you prefer that? I used rem because the size of the logo is not related to the font-size. It's not like a padding around text that should change with the font size. Maybe i should add a class to the version number, because the `font-size: 10px` is used only there. What do you think? 🤔

Reply:
usually you want all the child elements to scale with the parent if you increase the parent font-size. why the version number? the parent of the logo is the `.credits` element?

Reply:
Ah, thats the problem 😄  i removed the font size from `.credits` it's not needed, sorry.